packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/DocumentExtraction.java

```diff
@@ -0,0 +1,17 @@
+package software.uncharted.terarium.hmiserver.models.dataservice;
+
+import java.io.Serializable;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Data
+@Accessors(chain = true)
+public class DocumentExtraction implements Serializable {
+
+	private String file_name;
```

Comment:
Because the parent object is passed through the snake case converter, you should be able to write this variables as
```
@JsonAlias("file_name")
	private String fileName;
```

to have it be a bit more java/javascript like.  This is true of `asset_type` as well.

Reply:
done!